ovsInit.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#Start OVS in the Contiv container
+
+set -euo pipefail
+
+if [[ $(lsmod | cut -d" " -f1 | grep -q openvswitch) -eq 1 ]]; then
+	modprobe openvswitch
+fi
+
+mkdir -p /var/run/openvswitch
+mkdir -p /var/contiv/log/
+
+sleep 2
+
+if [ -d "/etc/openvswitch" ]; then
+	if [ -f "/etc/openvswitch/conf.db" ]; then
+		echo "INFO: The Open vSwitch database exists"
+	else
+		echo "INFO: The Open VSwitch database doesn't exist"
+		echo "INFO: Creating the Open VSwitch database..."
+		ovsdb-tool create /etc/openvswitch/conf.db /usr/share/openvswitch/vswitch.ovsschema
+	fi
+else
+	echo "CRITICAL: Open vSwitch is not mounted from host"
+	exit 1
+fi
+
+echo "INFO: Starting ovsdb-server..."
+ovsdb-server --remote=punix:/var/run/openvswitch/db.sock \
+	--remote=db:Open_vSwitch,Open_vSwitch,manager_options \
+	--private-key=db:Open_vSwitch,SSL,private_key \
+	--certificate=db:Open_vSwitch,SSL,certificate \
+	--bootstrap-ca-cert=db:Open_vSwitch,SSL,ca_cert \
+	--log-file=/var/contiv/log/ovs-db.log -vsyslog:info -vfile:info \
+	--pidfile /etc/openvswitch/conf.db &
+OVSDB_PID=$!
+
+echo "INFO: Starting ovs-vswitchd"
+ovs-vswitchd -v --pidfile --detach --log-file=/var/contiv/log/ovs-vswitchd.log \
+	-vconsole:err -vsyslog:info -vfile:info &
+VSWITCHD_PID=$!
+
+retry=0
+while [[ $(ovsdb-client list-dbs | grep -c Open_vSwitch) -eq 0 ]]; do
+	if [[ ${retry} -eq 5 ]]; then
+		echo "CRITICAL: Failed to start ovsdb in 5 seconds."
+		exit 1
+	else
+		echo "INFO: Waiting for ovsdb to start..."
+		sleep 1
+		((retry += 1))
+	fi
+done
+
+echo "INFO: Setting OVS manager (tcp)..."
+ovs-vsctl set-manager tcp:127.0.0.1:6640
+
+echo "INFO: Setting OVS manager (ptcp)..."
+ovs-vsctl set-manager ptcp:6640
+
+STATUS=0
+
+for pid in "$OVSDB_PID $VSWITCHD_PID"; do
+	wait $pid || let STATUS=1
```

Comment:
so this will keep wait until `$OVSDB_PID` dead and then start wait for `$VSWITCHD_PID`, if vswitch died but ovsdb still exist, it will still blocking 
I think we should have other "monitor alike" command to monitor these two pid, and exit when any of them died